Query not being parsed correctly

Seems like the latest release introduced a bug:
```
SELECT `id` FROM `public`.`test_table`
Normalized query:
test_table
2021-09-30 09:32:30.030 [vert.x-worker-thread-10] [DEBUG] {ResponseWriter:45} - Start Time=1632994350871, Timeout=-1000, Max Block Size=65505
2021-09-30 09:32:30.030 [vert.x-eventloop-thread-0] [ERROR] {JdbcBridgeVerticle:320} - Failed to respond
java.lang.IllegalArgumentException: At least one column is needed.
        at ru.yandex.clickhouse.jdbcbridge.core.TableDefinition.<init>(TableDefinition.java:86)
        at ru.yandex.clickhouse.jdbcbridge.core.TableDefinition.fromString(TableDefinition.java:371)
        at ru.yandex.clickhouse.jdbcbridge.core.QueryParser.getTable(QueryParser.java:118)
        at ru.yandex.clickhouse.jdbcbridge.JdbcBridgeVerticle.handleQuery(JdbcBridgeVerticle.java:473)
        at ru.yandex.clickhouse.jdbcbridge.internal.vertx.ext.web.impl.BlockingHandlerDecorator.lambda$handle$0(BlockingHandlerDecorator.java:48)
        at ru.yandex.clickhouse.jdbcbridge.internal.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:313)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at ru.yandex.clickhouse.jdbcbridge.internal.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.lang.Thread.run(Thread.java:748)

```

Happens on v2.0.5, using postgres driver 42.2.24 

Downgrading to v2.0.4 solved the issue

--- a/pom.xml
+++ b/pom.xml
@@ -5,7 +5,7 @@
     <groupId>ru.yandex.clickhouse</groupId>
     <artifactId>clickhouse-jdbc-bridge</artifactId>
     <packaging>jar</packaging>
-    <version>2.0.4-SNAPSHOT</version>
+    <version>2.0.6-SNAPSHOT</version>
     <name>ClickHouse JDBC Bridge</name>
     <url>https://github.com/ClickHouse/clickhouse-jdbc-bridge</url>
     <description>JDBC bridge for ClickHouse.</description>
@@ -13,8 +13,8 @@
 
     <properties>
         <caffeine.version>2.9.2</caffeine.version>
-        <dnsjava.version>3.4.1</dnsjava.version>
-        <vertx.version>3.9.8</vertx.version>
+        <dnsjava.version>3.4.2</dnsjava.version>
+        <vertx.version>3.9.9</vertx.version>
         <micrometer.version>1.1.19</micrometer.version>
         <slf4j.version>1.7.32</slf4j.version>
         <hikaricp.version>3.4.5</hikaricp.version>

--- a/src/main/java/ru/yandex/clickhouse/jdbcbridge/core/QueryParser.java
+++ b/src/main/java/ru/yandex/clickhouse/jdbcbridge/core/QueryParser.java
@@ -38,7 +38,8 @@ public class QueryParser {
     // Otherwise NULLs will be substituted with default values.
     // Currently supported only for 'mysql' table function.
     private static final String PARAM_EXT_TABLE_USE_NULLS = "external_table_functions_use_nulls";
-    private static final String PARAM_COLUMNS = "sample_block";
+    private static final String PARAM_COLUMNS = "columns";
+    private static final String PARAM_SAMPLE_BLOCK = "sample_block";
     private static final String PARAM_QUERY = "query";
     private static final String PARAM_DB_NAME = "db_name";
     private static final String PARAM_TABLE_NAME = "table_name";
@@ -260,8 +261,12 @@ public class QueryParser {
 
         String uri = Objects.requireNonNull(resolver).resolve(req.getParam(PARAM_CONNECTION_STRING));
         if (forWrite) {
-            query = new QueryParser(uri, req.getParam(PARAM_DB_NAME), req.getParam(PARAM_TABLE_NAME),
-                    req.getParam(PARAM_COLUMNS), req.getParam(PARAM_FORMAT_NAME), null, null);
+            String columns = req.getParam(PARAM_SAMPLE_BLOCK);
+            if (columns == null) {
+                columns = req.getParam(PARAM_COLUMNS);
+            }
+            query = new QueryParser(uri, req.getParam(PARAM_DB_NAME), req.getParam(PARAM_TABLE_NAME), columns,
+                    req.getParam(PARAM_FORMAT_NAME), null, null);
         } else {
             String schema = req.getParam(PARAM_SCHEMA);
             String table = req.getParam(PARAM_TABLE);
@@ -287,7 +292,10 @@ public class QueryParser {
             }
 
             if (columns == null) {
-                columns = req.getParam(PARAM_COLUMNS);
+                columns = req.getParam(PARAM_SAMPLE_BLOCK);
+                if (columns == null) {
+                    columns = req.getParam(PARAM_COLUMNS);
+                }
             }
 
             if (schema == null) {

